common/scylla-image-post-start.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Scylla cloud image post start service
+After=scylla-server.service
+ConditionPathExists=!/etc/scylla/machine_image_post_start
```

Comment:
Not really important, but on scylla-image-setup.service the file is `/etc/scylla/machine_image_configured`, so let's align this one to `/etc/scylla/machine_image_post_start_configured`.